linux_os/guide/system/accounts/accounts-banners/gui_login_banner/dconf_gnome_login_banner_text/rule.yml

```diff
@@ -7,7 +7,7 @@ title: 'Set the GNOME3 Login Warning Banner Text'
 description: |-
     In the default graphical environment, configuring the login warning banner text
     in the GNOME Display Manager's login screen can be configured on the login
-    screen by setting <tt>banner-message-text</tt> to <tt>string '<i>APPROVED_BANNER</i>'</tt>
+    screen by setting <tt>banner-message-text</tt> to <tt><i>APPROVED_BANNER</i>'</tt>
```

Comment:
I see only one apostrophe here, is that intentional?

Reply:
Thanks for noticing. I have fixed by rebase.